### PR TITLE
Alphabetical order YAML script for Istio org members

### PR DESCRIPTION
**Pull request by ronitblenz**
## I want to join the Istio org!

- [X] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [X] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [X] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [X] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [X] I have joined [Istio's Slack workspace](https://slack.istio.io)
- [X] I have written my company name or "Individual", if not affliated with a company, here: Individual
- [X] I have provided a link to at least one PR that I have successfully pushed to one of the Istio repos, here: 

Resolves Issue #1092 

Approach 👍 

I have made a directory called members, where you can add your name randomly (so that if somebody messes up with the already existing order, it would not affect the main members.yaml file). Now the script sorts the YAML and makes a copy of the ```tempname.yaml``` to generate updated ```members.yaml```.

Screenshot 2023-06-30 at 1 27 53 PM

### CODE REVIEW

File: org/members.yaml
Comment (craigbox) [inline org/members.yaml:1]: To add yourself as a member, please send a different PR.

**Review by craigbox — CHANGES_REQUESTED** [review]

**Comment by ericvn** [comment]
Can we not check in the temp_name.yaml and only generate it as needed to do the sort?

**Comment by ronitblenz** [comment]
Ok let me make the required changes.

**Comment by craigbox** [comment]
Hi Ronit,

Thanks for your contribution.  I'm going to close this PR as it has sat here for over 6 months.  If you're interested in revisiting this, I suggest you join the Istio Slack and check in with Eric or John in #contributors as to how you should change your code to best achieve this goal.

Good luck as you ramp up your contributions to open source.

**Pull request closed without merging**